src/monthly-timeseries/MonthlyTimeseries.js

```diff
@@ -0,0 +1,82 @@
+import PropTypes from "prop-types";
+import React from "react";
+import ResponsiveOrdinalFrame from "semiotic/lib/ResponsiveOrdinalFrame";
+import styled from "styled-components";
+import { THEME } from "../constants";
+import { formatAsPct } from "../utils";
+
+const MonthlyTimeseriesWrapper = styled.div`
+  height: 350px;
+  width: 100%;
+
+  /* classes provided by Semiotic */
+  .frame {
+    .axis-baseline {
+      stroke: ${(props) => props.theme.colors.chartAxis};
+    }
+
+    .axis-label,
+    .ordinal-labels {
+      fill: ${(props) => props.theme.colors.chartAxis};
+      font: ${(props) => props.theme.fonts.body};
+      font-size: 12px;
+    }
+
+    .background-graphics,
+    .visualization-layer {
+      shape-rendering: crispEdges;
+    }
```

Comment:
Is this the sanctioned approach to updating this visualization's styles?  If not, this is just something we'll need to keep in mind if these class names are internal to Semiotic and thus subject to change.  If nothing else, nice work on getting things to look the way we want them to 👍

Reply:
it is sanctioned insofar as there is not a universal API for adding arbitrary classes and styles to everything in Semiotic (and they do something similar in the official docs ... it's pretty conventional if not universal among charting libraries because of some underlying D3 conventions) ... I can't really say how stable these particular classes are but I don't think they are too esoteric, they are the major structural elements of every "frame" in Semiotic-land